projects/plugins/boost/jetpack-boost.php

```diff
@@ -68,7 +68,9 @@
 if ( isset( $_SERVER['REQUEST_URI'] ) ) {
 	// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
 	$request_path = explode( '?', wp_unslash( $_SERVER['REQUEST_URI'] ) )[0];
-	if ( '/_static/' === substr( $request_path, -9, 9 ) ) {
+
+	$static_prefix = defined( 'JETPACK_BOOST_STATIC_PREFIX' ) ? JETPACK_BOOST_STATIC_PREFIX : '/_jb_static/';
```

Comment:
Is there a reason to avoid using `jetpack_boost_get_static_prefix()` here?

Reply:
Yes. Jetpack Boost doesn't load code from modules which aren't enabled by default, so it would be unusual to load the Minify CSS/JS modules this early in the plugin life-cycle.

However, this particular bit of code wants to make a decision early about letting the minify service handle a given request.

So it made sense in this one case to handle it in-line before loading anything extra.